### DISCUSSION TITLE
PickPhoto from MediaPicker shows app name and black toolbar on Android

### DISCUSSION THREAD

**Discussion by KarlSearl** [dotnet/maui#26514] [Q&A]
Hi,

Can anyone explain a way of setting the toolbar colour and the text, I dont really want consumer the see the black bar and app name text. 

![pickphotoIssue](https://github.com/user-attachments/assets/5e1b1b6f-531e-4744-bcc4-df1a2bd570fb)

private async void TakePhoto(object sender, EventArgs e)
{
  FileResult photo = await MediaPicker.Default.PickPhotoAsync(new MediaPickerOptions
  {
    Title = "Select your photo"
  });
                  
  // Here, add the code that is being explained in the next step.
                   
}

**Comment by Yu-Core** [comment]
Use https://github.com/dimonovdd/Xamarin.MediaGallery

**Comment by logicielity** [comment]
> Hi,
> 
> Can anyone explain a way of setting the toolbar colour and the text, I dont really want consumer the see the black bar and app name text.
> 
> ![pickphotoIssue](https://private-user-images.githubusercontent.com/64684177/394399974-5e1b1b6f-531e-4744-bcc4-df1a2bd570fb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc5NTQ0NjEsIm5iZiI6MTczNzk1NDE2MSwicGF0aCI6Ii82NDY4NDE3Ny8zOTQzOTk5NzQtNWUxYjFiNmYtNTMxZS00NzQ0LWJjYzQtZGYxYTJiZDU3MGZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI3VDA1MDI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzYjI5MThiYTQ0Y2IzN2EyMDQ2MWRjN2ZmZDI5NmI3MDc5NDY0M2Q2MDVkNmU0ZDE4MjdiOTdhZWI3NjRkMDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.D335CNDODdCd6iixD_KXPqYYwN_dlT0w9qZBWZWyY5k)
> 
> private async void TakePhoto(object sender, EventArgs e) { FileResult photo = await MediaPicker.Default.PickPhotoAsync(new MediaPickerOptions { Title = "Select your photo" });
> 
> // Here, add the code that is being explained in the next step.
> 
> }

Did you manage to solve this?
